inscriptions/ISic000548.xml

```diff
@@ -202,59 +207,36 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR138306" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#GC">
                 <ab>
-                    <lb n="1"/>
-                    <expan>
-                        <abbr>D</abbr>
-                        <ex>is</ex>
-                    </expan>
-                    <expan>
-                        <abbr>M</abbr>
-                        <ex>anibus</ex>
-                    </expan>
-                    .
-                    <lb n="2"/>
-                    <expan>
-                        <abbr>Tac</abbr>
-                        <ex>itio</ex>
-                    </expan>
-                    Iulia<lb n="3" break="no"/>no
-                    <choice>
-                        <sic>
-                            <expan>
-                                <abbr>evok</abbr>
-                                <ex>ato</ex>
-                            </expan>
-                        </sic>
-                        <corr>evocato</corr>
-                    </choice>
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <expan><abbr>m</abbr><ex>anibus</ex></expan>
+                    <lb n="2"/><name><expan><abbr>Tac</abbr><ex>itio</ex></expan></name> <g ref="#interpunct">·</g> <name>Iulia
+                    <lb n="3" break="no"/>no</name> <g ref="#interpunct">·</g> <choice><sic><expan><abbr>evok</abbr><ex>ato</ex></expan></sic><corr>evocato</corr></choice>
```

Comment:
I would encode the variation using 'orig' and 'reg', rather than 'sic' and 'corr' - we don't wish to suggest it's an error or aberration; rather it is a variant form, simply using k instead of c. Also, we need to keep the two forms encoded in parallel, not suggest that the corrected form would not have been abbreviated, so:
`<expan><choice><orig><abbr>evok</abbr></orig><reg><abbr>evoc</abbr></reg></choice><ex>ato</ex></expan>`